package.json

```diff
@@ -146,7 +146,6 @@
     }
   },
   "dependencies": {
-    "skeleton-buffer": "file:./skeleton-buffer",
```

Comment:
Should we also delete the skeleton buffer folder and files as part of this change?